src/cephadm/cephadm

```diff
@@ -3351,11 +3345,6 @@ class CephContainer:
                 '--privileged',
                 # let OSD etc read block devs that haven't been chowned
                 '--group-add=disk'])
-        if self.ptrace and not self.privileged:
-            # if privileged, the SYS_PTRACE cap is already added
-            # in addition, --cap-add and --privileged are mutually
-            # exclusive since podman >= 2.0
-            cmd_args.append('--cap-add=SYS_PTRACE')
```

Comment:
this is a problem: we need to somehow only add SYS_PTRACE, if we're not privileged. But we could implement this logic elsewhere, for example in CephContainer.run_cmd

Reply:
> But we could implement this logic elsewhere, for example in CephContainer.run_cmd
This is a bit confusing. Aren't we already in `CephContainer.run_cmd()` here ?